src/network/connection.cpp

```diff
@@ -118,7 +118,8 @@ bool Connection::beginBundle() {
 	memset(m_BundledPacket, 0, sizeof(m_BundledPacket));
 	m_BundlePacketInnerCount = 0;
 
-	MUST_TRANSFER_BOOL(m_ServerFeatures.has(ServerFeatures::PROTOCOL_BUNDLE_SUPPORT));
+	MUST_TRANSFER_BOOL(m_ServerFeatures.has(EServerFeatureFlags::PROTOCOL_BUNDLE_SUPPORT
+	));
```

Comment:
Why?

Reply:
wdym "Why?"?

Reply:
to turncat the "));" i know autoformater.